pipeline.groovy

```diff
@@ -0,0 +1,10 @@
+import org.dotnet.ci.pipelines.Pipeline
+
+def windowsPipeline = Pipeline.createPipeline(this, 'build/buildpipeline/windows.groovy')
+def linuxPipeline = Pipeline.createPipeline(this, 'build/buildpipeline/linux.groovy')
+def osxPipeline = Pipeline.createPipeline(this, 'build/buildpipeline/osx.groovy')
+String configuration = 'Release'
+
+windowsPipeline.triggerPipelineOnPush(['Configuration':configuration])
```

Comment:
Did you not want this for PRs?

Reply:
Yes. Do I need to call `triggerPipelineOnEveryGithubPR`?

Reply:
Yes